components/auth/Join.tsx

```diff
@@ -27,7 +27,9 @@ const Join = () => {
       email: Yup.string().required().email(),
       password: Yup.string().required().min(8),
       team: Yup.string().required().min(3),
-      agreeToTerms: Yup.boolean().oneOf([true], 'You must agree to the Terms and Conditions.'),
+      agreeToTerms: process.env.NEXT_PUBLIC_TERMS_AND_CONDITIONS_URL
+        ? Yup.boolean().oneOf([true], 'You must agree to the Terms and Conditions.')
```

Comment:
Adding conditional statement here.
if NEXT_PUBLIC_TERMS_AND_CONDITIONS_URL is defined then agreeToTerms has to be true
else agreeToTerms can be a boolean which is initially defined as false